### DISCUSSION TITLE
天翼云盘挂载后经常无法访问

### DISCUSSION THREAD

**Discussion by LaurenceWang0527** [AlistGo/alist#3872] [Q&A]
请教大神
failed get link: Get "https://download.cloud.189.cn/file/downloadFile.action?dt=51&expired=1679057755032&sk=ac93a410-2647-4c97-bbc4-ec32c7cd4c70&ufi=32480150323584021&zyc=5&token=cloud3&sig=Mk4tLWOSMeNHC50Vj3GK%2FacZEkk%3D": dial tcp [************************************]:443: connect: network is unreachable
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/118277595/225909713-a49681af-0fcb-411f-9368-abd09289d46f.png">

**Comment by LaurenceWang0527** [comment]
挂载的是天翼云盘客户端

**Comment by LaurenceWang0527** [comment]
111

**Reply by kokikoku** [reply]
点击编辑，把不使用OCR 按钮打开。
可以看到家庭云里的内容了！
